Dockerfile

```diff
@@ -11,32 +11,29 @@ ENV PATH=${PATH}:/conda/bin
 # Enables "source activate conda"
 SHELL ["/bin/bash", "-c"]
 
-# Setup cuDF
-WORKDIR /root/cudf
-RUN git clone https://github.com/gaohao95/cudf.git /root/cudf \
-    && git checkout select-hash-functions \
-    && git submodule update --init --remote --recursive \
-    && conda env create --name cudf_dev --file conda/environments/cudf_dev_cuda11.0.yml \
-    && source activate cudf_dev \
-    && conda install -c rapidsai -c nvidia -c conda-forge -c defaults -y nccl \
-    && mkdir -p cpp/build \
-    && cd cpp/build \
-    && cmake .. -DCMAKE_INSTALL_PREFIX=${CONDA_PREFIX} -DGPU_ARCHS="70;80" \
-    && make -j install \
-    && conda clean -a -y
-ENV CUDF_HOME=/conda/envs/cudf_dev
+# Setup cuDF and NCCL
+WORKDIR /root
+RUN conda create --name cudf_release \
+    && source activate cudf_release \
+    && conda install -c rapidsai-nightly -c nvidia -c conda-forge -c defaults \
+        cudf=0.17 \
+        python=3.8 \
+        cudatoolkit=11.0 \
+        nccl \
+    && conda clean --all
```

Comment:
Previously you had `-y`.  Now you have removed it.

I tested this command without `-y` and it was interactive.  So I expect that this Dockerfile won't work.

Did you test this?

Reply:
Resovled in https://github.com/rapidsai/distributed-join/pull/48/commits/3e5a77979244758264783aebbf4cb3cba500c562.